src/scripts/main.js

```diff
@@ -1 +1,14 @@
 'use strict';
+
+const largeImg = document.getElementById('largeImg');
+const thumbsList = document.getElementById('thumbs');
+
+thumbsList.addEventListener('click', (e) => {
+  e.preventDefault();
```

Comment:
The `preventDefault()` method is called on every click event, which might not be necessary if the target is not an anchor element. Consider moving `preventDefault()` inside the `if (target)` block to ensure it only prevents the default action when an anchor element is clicked.